public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -69,7 +69,7 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
 
   useLayoutEffect(() => {
     if (isAscending && logsContainerRef.current) {
-      setScrollTop(logsContainerRef.current.offsetHeight);
+      setScrollTop(logsContainerRef.current.scrollHeight);
     } else {
       setScrollTop(0);
     }
```

Comment:
Does this work on both ascending and descending log panels?

Reply:
Hey @bennbollay! The last changes to the scroll feature is in [the iframe integration branch ](https://github.com/fusebit/grafana/pull/6 ). I asked to @MikeFuster to move only the scroll code to this branch so we can test and debug it atomically! It should be done in a brief 👍